static/_app/tools/idle-idle-gamedev/css/baseStyle.css

```
@@ -0,0 +1,515 @@
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+}
+.loadingLogo{
+    height: 50%;
+    max-width: 90%;
+}
+.luts-header{
+    padding-left: 0.5%;
+    width: 100%;
+    background: linear-gradient(0,darkcyan,cadetblue);
```

Biome (2.1.2)

[error] Unexpected nonstandard direction

You should fix the direction value to follow the syntax.
See MDN web docs for more details.

(lint/correctness/noInvalidDirectionInLinearGradient)